core/templates/components/state-editor/state-skill-editor/state-skill-editor.directive.ts

```diff
@@ -53,16 +53,23 @@ angular.module('oppia').directive('stateSkillEditor', [
           var _init = function() {
             TopicsAndSkillsDashboardBackendApiService.fetchDashboardDataAsync()
               .then(function(response) {
-                // CategorizedSkills: a dict of categorized ShortSkillSummary.
-                // UntriagedSkillSummaries:  array of Untriaged SkillSummary.
+                /**
+                 * CategorizedSkills: dict.
```

Comment:
This could still be clearer. Write a clear explanation and include the structure in the relevant part of the explanation. E.g.:

```
// categorizedSkills has the following format:
//    ...
// It represents {{description in prose}}.
```

Reply:
Done. PTAL!